lib/fog/aws/requests/dynamodb/query.rb

```diff
@@ -25,15 +23,17 @@ class Real
         #     * 'Count'<~Integer> - number of items in response
         #     * 'Items'<~Array> - array of items returned
         #     * 'LastEvaluatedKey'<~Hash> - last key scanned, can be passed to ExclusiveStartKey for pagination
-        def query(table_name, hash_key, options = {})
+        #
+        # See DynamoDB Documentation: http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Query.html
+        #
+        def query(table_name, options = {})
```

Comment:
Hmm. Maybe we should still catch that case to provide a more explicit/specific error message, even if we just have to bail out at that point. What do you think?